src/assets/fabricator/scripts/fabricator.js

```diff
@@ -96,64 +96,31 @@ fabricator.buildColorChips = function () {
 fabricator.setActiveItem = function () {
 
 	/**
-	 * @return {Array} Sorted array of menu item 'ids'
+	 * Match the window location with the menu item, set menu item as active
 	 */
-	var parsedItems = function () {
+	var setActive = function () {
 
-		var items = [],
-			id, href;
+		// get current file and hash without first slash
+		var current = (window.location.pathname + window.location.hash).replace(/^\//g, ''),
```

Comment:
add `|| index.html` to the end so that it matches the root page without the full path. e.g. `http//localhost:300/`